composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "reload/bugyield",
+  "description": "A script to sync between Harvest and Jira.",
+  "license": "MIT",
+  "autoload" : {
+    "psr-4": {
+      "": "src/"
+    }
+  },
+  "require": {
+    "php": ">=5.4.0",
+    "gridonic/hapi": "dev-master",
+    "symfony/console": "2.7.2",
+    "symfony/yaml": "2.7.2",
+    "alexkovalevych/jira-api": "1.0.5"
```

Comment:
Is there any reason why we lock this to patch releases?

Symfony packages follow semantic versioning pretty rigorously so I would think that ~2.7 would be sufficient.